flytekit/clis/sdk_in_container/run.py

```diff
@@ -819,21 +933,34 @@ def _create_command(
 
         # Add options for each of the workflow inputs
         params = []
-        for input_name, input_type_val in loaded_entity.python_interface.inputs_with_defaults.items():
+        for (
```

Comment:
why do we need the parentheses here

Reply:
The parentheses should be removed. Thanks!